apps/examples/testcase/le_tc/drivers/Kconfig

```diff
@@ -0,0 +1,53 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+menuconfig EXAMPLES_TESTCASE_DRIVERS
+	bool "Drivers TestCase Example"
+	default n
+	---help---
+		Enable the Drivers TestCase Example
+
+if EXAMPLES_TESTCASE_DRIVERS
+config TC_DRV_ALL
+	bool "All"
+	default n
+	select TC_DRIVERS_NULL
+	select TC_DRIVERS_ZERO
+	select TC_DRIVERS_RTC
+	select TC_DRIVERS_PWM
+	select TC_DRIVERS_WATCHDOG
+	select TC_DRIVERS_LOOP
+	select TC_DRIVERS_ADC
+
+config TC_DRIVERS_NULL
+	bool "Dev Null"
+	default n
+
+config TC_DRIVERS_ZERO
+	bool "Dev Zero"
+	default n
+
+config TC_DRIVERS_RTC
+	bool "Dev Rtc"
+	default n
+
+config TC_DRIVERS_PWM
+        bool "Dev PWM"
+        default n
+
+config TC_DRIVERS_WATCHDOG
+	bool "Dev Watchdog"
+	default n
+
+config TC_DRIVERS_LOOP
+	bool "Dev Loop"
+	default n
+
+config TC_DRIVERS_ADC
+	bool "Dev ADC"
```

Comment:
is "depends on ADC" not needed?

Reply:
Done

Reply:
This is not only for s5j. I think CONFIG_ADC is better..

Reply:
Done